### PR TITLE
Message thread

### PR DESCRIPTION
**Pull request by ahr1384**
image

### CODE REVIEW

**Comment by ahr1384** [comment]
plc check it @fabio-ivona

File: src/Concerns/HasBotsAndChats.php [hunk 1]
```diff
@@ -56,6 +56,11 @@ public function chat(TelegraphChat|string $chat): Telegraph
         return $telegraph;
     }
 
+    public function thread(int $thread): Telegraph
```
Comment (fabio-ivona) [inline src/Concerns/HasBotsAndChats.php:59]: @ahr1384 maybe this could be a more expressive API as `->onThread(int $thread)`, what do you think?

**Comment by fabio-ivona** [comment]
@ahr1384 thanks for your contribution 🙌

Would you mind to add tests for this feature?

**Comment by ahr1384** [comment]
Hi @fabio-ivona,
I tested it in my local app.
But it's a simple change and obviously, there's no problem, given that the **withData** function already exists and I use it in another function.

**Comment by fabio-ivona** [comment]
@ahr1384 I know it is a trivial change, but every new feature is needed to be tested because someone else in future may break it in some way, tests avoid this cases

**Comment by ahr1384** [comment]
Well, I couldn't test for this, any advice on which section it should be in? And I think I have a problem with the PHP version.

**Comment by fabio-ivona** [comment]
You could place the test in HasBotsAndChatsatest.php

**Comment by fabio-ivona** [comment]
@ahr1384 any news here? have you managed to write tests?

also, would you mind to rebase from main? there are some conflicts in docs files

**Pull request closed without merging**